README.md

```diff
@@ -1776,6 +1784,8 @@ hz.getSet('my-distributed-set').then(function (s) {
 });
 ```
 
+Hazelcast Set uses `ItemListener` to listen to events that occur when items are added to and removed from the Set. See the [Item Listener section](#7523-item-listener) for information on how to create an item listener object and register it.
```

Comment:
Hazelcast Set uses `ItemListener` to listen to the events that occur when the items are added to or removed from the Set.